plans/feat-map-plugin-1227.md

```
@@ -11,259 +24,196 @@ Tracking issue: [#1227](https://github.com/receptron/mulmoclaude/issues/1227)
-```markdown
----
-title: お気に入りの寿司屋
-coords:
-  lat: 35.6762
-  lng: 139.6503
-mapZoom: 16             # optional, default 16
----
+### Wiring topology
 
-レビュー本文...
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)